### PR TITLE
Update Homepage Callout Copy and Logo

### PR DESCRIPTION
**Pull request by backlineint**
We don't talk about Bruno... I mean hosting.

### CODE REVIEW

File: starters/next-drupal-starter/pages/index.js [hunk 1]
```diff
@@ -28,9 +28,9 @@ export default function Home({ articles, hrefLang, multiLanguage }) {
 
           <div className="text-2xl">
             <div className="bg-black text-white rounded flex items-center justify-center p-4">
-              Decoupled hosting by{" "}
+              Decoupled Drupal on{" "}
               <Image
-                src="/pantheon.svg"
+                src="/pantheon.png"
```
Comment (CobyPear) [inline starters/next-drupal-starter/pages/index.js:33]: The fallback image below (line 72) and on the /articles page also needs the .png switch

**Review by CobyPear — CHANGES_REQUESTED** [review]
Just need to update the other spots in the next starter where the svg was being used.

If you want to take this opportunity to update the favicons, there's one in `web/static/images`. If not, I have that on another PR.

File: starters/next-drupal-starter/pages/index.js (hunk 1 above)
Comment (backlineint) [reply starters/next-drupal-starter/pages/index.js:33]: Whoops, I think I got the rest of them.

**Comment by backlineint** [comment]
> Just need to update the other spots in the next starter where the svg was being used.
> 
> If you want to take this opportunity to update the favicons, there's one in `web/static/images`. If not, I have that on another PR.

Other PR works for me.

**Review by CobyPear — APPROVED** [review]
👍

**Pull request merged**